Missing library requirement for install

When installing py3dtilers, I get the following error:

```
Building wheel for psycopg2 (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/admindc/py3dtilers/venv/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-4_wtozrx/psycopg2/setup.py'"'"'; __file__='"'"'/tmp/pip-install-4_wtozrx/psycopg2/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-1gn4m7td
       cwd: /tmp/pip-install-4_wtozrx/psycopg2/
  Complete output (38 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.9
  creating build/lib.linux-x86_64-3.9/psycopg2
  copying lib/extras.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/__init__.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/_ipaddress.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/sql.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/pool.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/tz.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/errors.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/_range.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/_json.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/extensions.py -> build/lib.linux-x86_64-3.9/psycopg2
  copying lib/errorcodes.py -> build/lib.linux-x86_64-3.9/psycopg2
  running build_ext
  building 'psycopg2._psycopg' extension
  creating build/temp.linux-x86_64-3.9
  creating build/temp.linux-x86_64-3.9/psycopg
  x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DPSYCOPG_VERSION=2.9.6 (dt dec pq3 ext lo64) -DPSYCOPG_DEBUG=1 -DPG_VERSION_NUM=130010 -DHAVE_LO64=1 -DPSYCOPG_DEBUG=1 -I/home/admindc/py3dtilers/venv/include -I/usr/include/python3.9 -I. -I/usr/include/postgresql -I/usr/include/postgresql/13/server -I/usr/include/libxml2 -c psycopg/adapter_asis.c -o build/temp.linux-x86_64-3.9/psycopg/adapter_asis.o -Wdeclaration-after-statement
  In file included from psycopg/adapter_asis.c:28:
  ./psycopg/psycopg.h:35:10: fatal error: Python.h: No such file or directory
     35 | #include <Python.h>
        |          ^~~~~~~~~~
  compilation terminated.
  
  It appears you are missing some prerequisite to build the package from source.
 
  You may install a binary package by installing 'psycopg2-binary' from PyPI.
  If you want to install psycopg2 from source, please install the packages
  required for the build and try again.
  
  For further information please check the 'doc/src/install.rst' file (also at
  <https://www.psycopg.org/docs/install.html>).
  
  error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for psycopg2
```

Installing `python3.9-dev` fixes it.

--- a/README.md
+++ b/README.md
@@ -36,7 +36,7 @@ Install binary sub-dependencies with your platform package installer e.g. for Ub
 
 ```bash
 apt-get install -y libpq-dev       # required usage of psycopg2 within py3dtilers
-apt-get install python3.9          # Python3 version must be 3.9
+apt-get install python3.9 python3.9-dev         # Python3 version must be 3.9
 ```
 
 Install Py3DTilers in a safe [python virtual environment](https://docs.python.org/3/tutorial/venv.html) (not mandatory yet quite recommended)

